wbooks/src/app/screens/BookCard/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
```

Comment:
Bad folder structure, check [this](https://trello.com/c/xpQBgDdp/8-n-53-estructura-de-carpetas-y-convenciones) again!

This must be a component, not a screen! Then you can add a `Library` screen and use this component